feature-libs/organization/order-approval/assets/translations/en/order-approval.i18n.ts

```diff
@@ -17,6 +17,20 @@ export const orderApprovalDetails = {
       placeholder: '',
     },
   },
+  permissionResults: {
+    header: 'Customer Approval Details',
+    permission: 'Permission',
+    approver: 'Approver',
+    status: 'Status',
+    approverComments: 'Approver Comments',
+    noApprovalNotes: 'None',
+    permissionType_B2BBudgetExceededPermission:
+      'The budget associated with chosen cost center has been exceeded',
+    permissionType_B2BOrderThresholdPermission:
+      'Order total exceeded per-order limit',
+    permissionType_B2BOrderThresholdTimespanPermission:
+      'Order total exceeded per-timespan limit',
```

Comment:
Order total exceeded `the` per-timespan limit

Reply:
Done.